### PR TITLE
Turd distance

### PR DESCRIPTION
**Pull request by TarikGul**
Calculate the distance between a task and the user, added frontend util file, put a distance parser in frontend util

### CODE REVIEW

File: frontend/src/components/sidebar/active.jsx [hunk 1]
```diff
@@ -32,7 +32,8 @@ class ActiveSidebar extends React.Component {
                                         currentUserId={currentUserId}
                                         history={history} 
                                         activeTask={this.props.activeTask}
-                                        receiveActiveTaskId={this.props.receiveActiveTaskId}/>
+                                        receiveActiveTaskId={this.props.receiveActiveTaskId}
+                                        currentPosition={this.props.currentPosition}/>
```
Comment (emostov) [inline frontend/src/components/sidebar/active.jsx]: For future stuff, try using destructing for variables that are being pulled out of the same object, like props.

File: frontend/src/components/sidebar/card.jsx [hunk 2]
```diff
@@ -71,7 +95,7 @@ class Card extends React.Component {
                 <div className="card-header-container">
                     <FontAwesomeIcon className="fa-minus" icon={faMinus} />
                   <div className={"card-head-active"}>
-                    0.1 Miles Away
+                    {`${this.state.distance} miles away`}
```
Comment (emostov) [inline frontend/src/components/sidebar/card.jsx]: We should eventually do something here incase distance is null so it doesnt just say 'miles away'

File: frontend/src/components/sidebar/active.jsx (hunk 1 above)
Comment (TarikGul) [reply frontend/src/components/sidebar/active.jsx]: hell yea thanks my dude:)
will do illr refactor on my next push

File: frontend/src/components/sidebar/card.jsx (hunk 2 above)
Comment (TarikGul) [reply frontend/src/components/sidebar/card.jsx]: yes i agree i try to break it later and see what i can do to get around something like that

**Pull request merged**